packages/contracts-bedrock/test/kontrol/scripts/run-kontrol.sh

```diff
@@ -0,0 +1,258 @@
+#!/bin/bash
+set -euo pipefail
+
+#####################
+# Support Functions #
+#####################
+blank_line() { echo '' >&2 ; }
+notif() { echo "== $0: $*" >&2 ; }
+usage() {
+  echo "Usage: $0 [-h|--help] [container] [local] [dev]" 1>&2
```

Comment:
These options are exclusive it sounds like?
```suggestion
  echo "Usage: $0 [-h|--help] [container|local|dev]" 1>&2
```

Reply:
Also can you expand on when to use the local vs. container modes? I see they each have help text but it's not clear to me when I'd want to use each when running on my personal machine. `container` to debug CI failures but `local` otherwise? Why not always `container`?

Reply:
Absolutely! The reasoning behind these three options is the following:
- `container`: run on your machine what will be run on CI, using the docker image
    - Use case: CI debugging
- `local`: run on your machine without any container, forcing the project to specify the Kontrol version. 
    - Use case: Allow local reproducibility, keeping in mind less experienced Kontrol users who want to reproduce the proofs without a container but don't want to experiment with different Kontrol versions
- `dev`: run on your machine on your terms. That is, with whatever Kontrol configuration you have installed
    - Use case: Proof development. This option replaces `run-kontrol-local.sh`

Let me know if these make sense to you! I'll be documenting these in the readme

Reply:
In other words, the main reason for `local` is to prevent errors of the type "I tried running the proofs on my machine and they're not working" because of a bad Kontrol version

Reply:
Got it, thanks for the explanation! Would be great to clarify in the code comments or help text there